tools/percolator/metafiles2pin.xml

```diff
@@ -0,0 +1,110 @@
+<tool id="metafiles2pin" name="Merge search data" version="1.0">
+    <description>before creating percolator input</description>
+    <stdio>
+        <exit_code range="1:" />
+    </stdio>
+    <command interpreter="python">metafiles2pin.py
```

Comment:
interpreter is now deprecated, please use `python $__tool_directory__/metafiles2pin.py` instead